projects/plugins/social/src/class-rest-social-note-controller.php

```diff
@@ -66,6 +66,17 @@ public function update_post_shares( $request ) {
 
 		if ( $post && $post->post_type === Note::JETPACK_SOCIAL_NOTE_CPT && $post->post_status === 'publish' && isset( $post_meta[ self::SOCIAL_SHARES_POST_META_KEY ] ) ) {
 			update_post_meta( $post_id, self::SOCIAL_SHARES_POST_META_KEY, $post_meta[ self::SOCIAL_SHARES_POST_META_KEY ] );
+			$urls = array();
```

Comment:
Mental note to myself to deprecate this endpoint. The endpoint in the publicize package (projects/packages/publicize/src/class-rest-controller.php) is the one being used for post meta update.